guide/tests/message_transformation_test.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+import ballerina/test;
+
+boolean serviceStarted;
+
+function startService() {
+    serviceStarted = test:startServices("message_transformation");
+}
+
+@test:Config {
+    before: "startService",
+    after: "stopService"
+}
+
+function message_transformation_check() {
```

Comment:
Shall we rename the function name to testMessageTransformation().

Reply:
Done